packages/lwc-language-server/scripts/convert.js

```diff
@@ -50,26 +50,71 @@ const tags = Object.keys(data).map(key => {
     };
 });
 
+//make globalAttribute changes here, not in standard-lwc.json as they'll be overwritten
```

Comment:
OK, this is super nin-picky, but I thought out ES-lint rules required a space after the `//`

Reply:
Other than this, the PR looks good.

Reply:
I think something's amiss with eslint here; @gbockus-sf gave similar feedback on a previous PR in here! I will get that done in short order though 👍